frameworks/hdfs/src/main/dist/hdfs-site.xml

```diff
@@ -20,7 +20,7 @@
     {{#TLS_ENABLED}}
     <property>
         <name>dfs.http.policy</name>
-        <value>HTTP_AND_HTTPS</value>
+        <value>HTTPS_ONLY</value>
```

Comment:
Perhaps value should be templated in case TLS is not enabled? This looks like it might break if it's using HTTPS by default and the setup doesn't include TLS.

Reply:
Ah, never mind. This entire property is templated, meaning if TLS isn't enabled it'll use HTTP by default 👍

Reply:
This setting implies to me that HDFS can do HTTP, HTTP + HTTPS, or just HTTPS. Is that true? If true, I think we should support that as we've been doing so for the other services.

Reply:
Yeah, so setting this to HTTPS_ONLY in case of enabled TLS seems to be correct, as not setting it defaults to HTTP. Having `HTTP_AND_HTTPS` seems ambiguously too generous, as we either have TLS, or we don't, in which case this setting (and the default) provide sufficient coverage.